client/components/header-cake/back.jsx

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import ObserveWindowSizeMixin from 'lib/mixins/observe-window-resize';
+import Gridicon from 'components/gridicon';
+
+export default React.createClass( {
+
+	displayName: 'HeaderCakeBack',
+
+	mixins: [ ObserveWindowSizeMixin ],
+
+	propTypes: {
+		onClick: PropTypes.func,
```

Comment:
`onClick` was updated in the documentation above to be "required" but by not having it required here it becomes confusing. I would recommend we make the two sources of information match